.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
   build:
     strategy:
       matrix:
-        node: [18, 16]
+        node: [18, 20]
```

Comment:
lets keep all builds ( cjs, esm ) only on v18

Reply:
Initially, I used only node v18. But the the Unit Test failed for v20 as it depends on this build. As you can see here: https://github.com/web3/web3.js/actions/runs/7445821806/job/20254915012
For that I added the build with node v20 to CJS.